### PR TITLE
Porting owners file from kube-deploy

### LINKED ISSUE #3: Create owners file
Follow suit of https://github.com/kubernetes/kube-deploy/pull/481

Original context https://docs.google.com/a/heptio.com/document/d/16ils69KImmE94RlmzjWDrkmFZysgB2J4lGnYMRN89WM/edit?disco=AAAAB1xreHk

### PR DESCRIPTION
**Pull request by kris-nova**
Copying an owners file is just to get us started

closes #3

### CODE REVIEW

**Comment by kris-nova** [comment]
I think this is a weird PR because it will give the rest of the owners/maintainers merge rights.

CC @roberthbailey @timothysc @krousey

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+aliases:
+  sig-cluster-lifecycle-leads:
+    - lukemarsden
+    - luxas
+    - roberthbailey
+    - timothysc
+  cluster-api-admins:
+    - justinsb
+    - krousey
+    - luxas
```
Comment (krousey) [inline OWNERS_ALIASES]: I agree that he should be in the sig leads and part of approvers. I think he was a kube-deploy admin, but not very active in the cluster-api aspect of it. @luxas, does that seem accurate?

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-cluster-lifecycle-leads:
+    - lukemarsden
+    - luxas
+    - roberthbailey
+    - timothysc
+  cluster-api-admins:
```
Comment (krousey) [inline OWNERS_ALIASES]: Does this come with github repo privileges like being able to create labels and other things? Is this meant to mirror that structure? If so, then this list should mirror the github organizational structure. It's fine to discuss who should have that level of access here though and then mirror it back to github.

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - justinsb
```
Comment (roberthbailey) [inline OWNERS_ALIASES]: Justin was a kube-deploy admin for the aws image builder sub-project. Not sure if he cares about being an admin for this repo or not.

File: OWNERS_ALIASES (hunk 2 above)
Comment (roberthbailey) [reply OWNERS_ALIASES]: In the kube-deploy repo, this list came from the github group with the same name. If we aren't going to be setting up groups in this org then there isn't any point separating out admins from maintainers here.

**Comment by kris-nova** [comment]
Wondering if we can keep this PR as is (as it's a port) and make changes in other PRs?

In general I would tend to favor unchanged ports with later PRs for changes just to keep a clean history and reduce complexity.

**Comment by kris-nova** [comment]
I went ahead and merged this in - so at least we now have owners defined for work moving forward. Hope this works for everyone. 😬

**Comment by luxas** [comment]
To answer your question, I don't _need_ to be in the `cluster-api-admins` group there, but in the end there's not a difference as I'm a lead and with the current structure I'd get approving rights in any case.

**Pull request merged**